rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/20_translog.yml

```diff
@@ -77,3 +77,51 @@ setup:
       indices.stats:
         metric: [ translog ]
   - gte: { indices.test.primaries.translog.earliest_last_modified_age: 0 }
+
+---
+"Translog stats on closed indices":
+  - skip:
+      version: " - 7.2.99"
```

Comment:
I think we should skip this test until 7.9.99 then adjust to this value after backporting.

Reply:
weird that this did not fail the tests? Perhaps BWC is disabled?

Reply:
Pfff... I should have seen this. And yes, bwc tests were disabled at that time. I merged master and bwc are enabled again.